docs/streams/getting-started-streams.md

```diff
@@ -0,0 +1,454 @@
+    });
+      producer.on("open", () => {
+        // If your message is an object, convert the object to a string.
+        // e.g. const message = JSON.stringify({message:'Hello World'});
+        const message = "Hello World";
+        const payloadObj = { payload: Buffer.from(message).toString("base64") };
+        producer.send(JSON.stringify(payloadObj));
+      });
+      producer.on("message", (msg) => {
+        console.log(msg, "Sent Successfully");
+      });
+
+    } catch(e) {
+      await console.log("Publishing could not be done because "+ e);
+    }
+}
+
+streams()
+```
+
+</TabItem>
+<TabItem value="py" label="Python">
+
+```py
+producer = client.create_stream_producer(demo_stream, local=prefixBool)
+for i in range(10):
+    msg1 = "Persistent Hello from " + "("+ str(i) +")"
+    data = {
+        "payload" : base64.b64encode(six.b(msg1)).decode("utf-8")
+    }
+    print("Stream: ", msg1)
+    print(producer.send(json.dumps(data)))
+```
+
+</TabItem>
+</Tabs>  
+
+### Step 5. Subscribe to Stream
+
+Example to subscribe documents from a stream. The stream can be either a local stream or a geo-replicated global stream.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="Javascript">
+
+```js
+async function getDCList() {
+  const geo_fabric = "_system"
+  let dcListAll = await client.listUserFabrics();
+  let dcListObject = await dcListAll.find(function(o) { return o.name === geo_fabric; });
+  return dcListObject.options.dcList.split(",");
+}
+
+(async function() {
+  const dcList = await getDCList();
+  await console.log("dcList: ", dcList);
+  await client.createStream("my-stream", true);
+  
+  //Here the last boolean value tells if the stream is local or global. false means that it is global.
+  // Publishing streams
+  const consumer = await client.createStreamReader("my-stream", "my-subscription", true);
+  consumer.on("message", (msg) => {
+    const { payload, messageId } = JSON.parse(msg);
+    
+    // Logging received message payload(ASCII encoded) to decode use atob()
+    console.log(payload);
+    // Send message acknowledgement
+    consumer.send(JSON.stringify({ messageId }));
+  });
+
+})();
+```
+
+</TabItem>
+<TabItem value="py" label="Python">
+
+```py
+subscriber = client.subscribe(stream=demo_stream, local=prefixBool,
+    subscription_name="test-subscription-1")
+for i in range(10):
+    print("In ",i)
+    m1 = json.loads(subscriber.recv())  # Listen on stream for any receiving messages
+    msg1 = base64.b64decode(m1["payload"])
+    print(F"Received message '{msg1}' id='{m1['messageId']}'") # Print the received message
+    subscriber.send(json.dumps({'messageId': m1['messageId']})) # Acknowledge the received message
+```
+
+</TabItem>
+</Tabs>
+
+## Full Demo File
+
+Copy the code block below into a file and run it in your favorite IDE.
+
+<Tabs groupId="operating-systems">
+<TabItem value="js" label="JavaScript">
+
+```js
+// Connect to GDN.
+const jsc8 = require("jsc8");
```

Comment:
I have sent the full demo file codes for both javascript (jsc8) and python (pyC8) on slack